src/input.cpp

```diff
@@ -131,39 +197,35 @@ void Input::Update() {
 	source->Update();
 	auto& pressed_buttons = source->GetPressedButtons();
 
-	if (recording_input) {
-		record_log << source->GetPressedNonSystemButtons() << '\n';
-	}
-
```

Comment:
The GetPressedNonSystemButtons line is lost

Reply:
This was moved into `Update()`, but it's not using the system buttons. Great catch, this was a rebase bug.

Reply:
"was a bug"? I still do not see it.

Reply:
The PR moved this to `Input::Source::Record` in `input_source.cpp`. The rebase bug was I didn't update that part to use `GetPressedNonSystemButtons()`.

The later added comment "Record only system buttons" fixes